aea/cli/utils/click_utils.py

```diff
@@ -148,6 +148,17 @@ def wrapper(f):
     return wrapper
 
 
+def registry_path_option(f):
+    """Add registry path aea option."""
+    return option(
+        "--registry-path",
+        "--alias",
```

Comment:
Not sure about `--alias`. That will be very confusing.

Reply:
sorry, forgot to remove. removed